test/e2e/case17_gatekeeper_sync_test.go

```diff
@@ -125,6 +208,205 @@ var _ = Describe("Test Gatekeeper ConstraintTemplate sync", Ordered, Label("skip
 			return compliance
 		}, defaultTimeoutSeconds, 1).Should(Equal("Compliant"))
 	})
+
+	It("should return Gatekeeper audit results", func() {
+		By("Checking if policy status is compliant for the constraint")
+		Eventually(func(g Gomega) {
+			managedPolicyUnstructured := propagatorutils.GetWithTimeout(
+				clientManagedDynamic, gvrPolicy, policyName, clusterNamespace, true, defaultTimeoutSeconds,
+			)
+
+			managedPolicy := policyv1.Policy{}
+			err := runtime.DefaultUnstructuredConverter.FromUnstructured(
+				managedPolicyUnstructured.Object, &managedPolicy,
+			)
+			g.Expect(err).To(BeNil())
+
+			g.Expect(managedPolicy.Status.Details).To(HaveLen(2))
+			g.Expect(managedPolicy.Status.Details[1].TemplateMeta.GetName()).To(Equal(gkConstraintName))
+			history := managedPolicy.Status.Details[1].History
+			g.Expect(len(history)).ToNot(Equal(0))
+			expectedMsg := "Compliant; The constraint has no violations"
+			g.Expect(history[0].Message).To(
+				Equal(expectedMsg),
+				fmt.Sprintf("Got %s but expected %s", history[0].Message, expectedMsg),
+			)
+		}, "120s", 1).Should(Succeed())
```

Comment:
Nit: Following other conventions here. Is there a reason to hardcode this value (as well as the other instances in this suite)?
```suggestion
		}, defaultTimeoutSeconds*4, 1).Should(Succeed())
```

Reply:
I changed this to use a different constant.